actions/pacman_action.go

```diff
@@ -0,0 +1,39 @@
+/*
+Pacman Action
+
+Install packages and their dependencies to the target rootfs with 'pacman'.
+
+	# Yaml syntax:
+	- action: pacman
+	  packages:
+	    - package1
+	    - package2
+
+Mandatory properties:
+
+- packages -- list of packages to install
+*/
+package actions
+
+import (
+	"github.com/go-debos/debos"
+)
+
+type PacmanAction struct {
+	debos.BaseAction `yaml:",inline"`
+	Packages         []string
+}
+
+func (p *PacmanAction) Run(context *debos.DebosContext) error {
+	p.LogStart()
+
+	pacmanOptions := []string{"pacman", "-Syu", "--noconfirm"}
```

Comment:
i think it'd be great to check in Verify if pacman is installed

But this can be a follow-up as I we should do the same check with all actions. Can you open an issue about that ?

Reply:
Greed - a handful of actions make this assumption. Opened https://github.com/go-debos/debos/issues/379